boards/ti/lp_mspm0g3507/board.cmake

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(openocd "--cmd-pre-load=reset init")
+# unprotect sectors before running
+board_runner_args(openocd "--cmd-pre-load=flash protect 1 0 last off")
+board_runner_args(openocd "--cmd-pre-load=flash erase_sector 1 0 last")
```

Comment:
We can leave out openOCD support for now if that's easier? It doesn't have public support so we're working on adding it at the moment

Reply:
Up to you but I think it's fine to keep it, just that I would group the script in a config file

Reply:
@msp-ti if OpenOCD support is not public, what would be the recommendation for someone trying to flash to the board? would `west flash` work ? also since these pre load commands are not Zephyr specific can that be part of your generic OpenOCD support?

Reply:
West Flash board can be achieved. Made note in the description